aleph-client/Cargo.toml

```diff
@@ -2,6 +2,8 @@
 name = "aleph_client"
 version = "1.4.0"
 edition = "2021"
+license = "Apache 2.0"
+license-file = "./LICENSE"
```

Comment:
General comment to the whole PR: do we need to specify each license separately as a file? https://doc.rust-lang.org/cargo/reference/manifest.html#the-license-and-license-file-fields specifies we don't need both `license` and `license-file` to be set unless we're dealing with a known license from the list https://spdx.org/licenses/ , which is true in our case.